mLRS/Common/hal/esp/tx-hal-radiomaster-bandit-micro-900-esp32.h

```diff
@@ -158,16 +166,108 @@ IRAM_ATTR bool button_pressed(void) { return false; }
 
 //-- LEDs
 
+
+
 #define LED_RED                   IO_P15
 
-void leds_init(void)
-{
-    gpio_init(LED_RED, IO_MODE_OUTPUT_PP_LOW);
-}
+#ifdef TX_ELRS_RADIOMASTER_BANDIT_900_ESP32
+
+    #include <NeoPixelBus.h>
```

Comment:
pl remove all the identation, just like normal code

Reply:
Done, cleaned up white space at the top too